package.json

```diff
@@ -22,10 +22,11 @@
     }
   ],
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=4"
   },
   "scripts": {
     "test": "xo && npm run prepublish && nyc ava",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
`flow check` is not the same thing though. `flow check` does a full check each time. `flow` is an alias for `flow status`. I'm looking for `flow status --no-error` (or something), so I don't have to have that bash exit code checking.